runners/direct-java/src/test/java/org/apache/beam/runners/direct/AvroIOShardedWriteFactoryTest.java

```diff
@@ -101,12 +103,18 @@ public void originalShardedToManyReturnsExplicitlySharded() throws Exception {
 
     assertThat(overridden, not(Matchers.<PTransform<PCollection<String>, PDone>>equalTo(original)));
 
-    TestPipeline p = TestPipeline.create();
+    Pipeline p = getPipeline();
     String[] elems = new String[] {"foo", "bar", "baz", "spam", "ham", "eggs"};
     p.apply(Create.<String>of(elems)).apply(overridden);
 
     file.delete();
     p.run();
     AvroIOTest.assertTestOutputs(elems, 3, file.getAbsolutePath(), original.getShardNameTemplate());
   }
+
+  private Pipeline getPipeline() {
```

Comment:
In case debugging of the build issues goes on, I am happy to pull just these sorts of fixes, which seem to stand on their own.